server/service/trigger.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 type triggerRequest struct {
-	Name string `query:"name"`
+	Name string `query:"name,optional"`
```

Comment:
When would you not send the name?

Reply:
`fleetctl trigger` by itself can be used to get the list of all trigger names